package.json

```diff
@@ -16,7 +16,7 @@
   ],
   "scripts": {
     "lint": "eslint .",
-    "test:unit": "mocha --recursive --extension test.js",
+    "test:unit": "mocha -r jsdom-global/register --recursive --extension test.js",
```

Comment:
Setting `global.window = {}` or some such doesn't appear to be sufficient in this case because of the way mocha imports dependencies. jsdom-global is a relatively quick and easy way to mock up globals like window/document without too much pain and suffering.